doc/source/getting_started/index.rst

```diff
@@ -3,65 +3,65 @@
 Getting Started
 ===============
 
-Please, carefully read all the instructions in this page in order to achieve a
-successfully install of ``ansys-templates``.
+To successfully install ``pyansysansys-templates``, read all instructions on this page carefully.
 
 
 Before Installing
 -----------------
 
-Several requirements need to be fulfilled before you can proceed installing
-``ansys-templates`` tool.
+Several requirements must be met before you install
+``pyansys-templates``.
 
 
-Upgrading pip
+Upgrading Pip
 ^^^^^^^^^^^^^
 
-Before installing the ``ansys-templates``, it is advisable to upgrade `pip`_ by
-running:
+Upgrade `pip`_ with:
 
 .. code:: bash
 
    python -m pip install --upgrade pip
 
 
-Installing pipx
+Installing Pipx
 ^^^^^^^^^^^^^^^
  
 The ``ansys-templates`` tool is built on top of Python. To ensure a clean
-installation you can use `pipx`_. This tool ensures an isolated installation of
-any Python tool you want to use. You can install it by running:
+installation, you can use `pipx`_. It ensures an isolated installation of
+any Python tool that you want to use. 
+
+Install `pipx`_ with:
 
 .. code:: bash
 
    python -m pip install --user pipx
 
-Ensure that `pipx`_ is in your ``PATH`` by running:
+Ensure that `pipx`_ is in your ``PATH`` with:
 
 .. code:: bash
 
    python -m pipx ensurepath
```

Comment:
Kelly had trouble here. She could run python -m pip ensurepath but not use pipx. It could be she missed something though as she is new to process.

Reply:
> python -m pip ensurepath

Just to make sure, this should be `python -m pipx ensurepath`  (not `pip`).

Reply:
That is what it says--but it didn't work for her.

Reply:
This error does not provide enough information... I recommend Kelly to open an issue on this repo pasting the full set of commands and the output she gets. If she is not familiar with this workflow, there is no problem on having a quick virtual meeting to investigating this @PipKat.